packages/server/adapters/aws-lambda/index.d.ts

```diff
@@ -1 +1 @@
-export * from '../../dist/adapters/aws-lambda';
+export * from '../../dist/adapters/aws-lambda/index';
```

Comment:
could escape the index part but it doesn't really hurt so keeping the script as minimal as possible